hibike/hibike_tester.py

```diff
@@ -1,4 +1,5 @@
 import hibike_process
+import time
```

Comment:
Personally agree with Doug on this, the time import seems useful for testing. For now I have not made a change